src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css

```diff
@@ -929,6 +929,10 @@ blockquote {
   background-color: var(--pst-color-background) !important;
 }
 
+.sd-hide-link-text {
+  display: none;
+}
```

Comment:
Could you indicate the purpose of this? Looks like this is using the prefix `sd` to indicate that it is a `sphinx-design` class.